src/sagas/wallet.js

```diff
@@ -194,11 +206,18 @@ export function* startWallet(action) {
     });
   } catch (e) {
     if (useWalletService) {
+      yield put(setShowWalletServiceLoadFailedModal(true));
+      yield take(types.WALLETSERVICE_RENDER_FAILED_ACK);
```

Comment:
But if a user dismiss the modal without tapping on "ok" button? Should we consider it an acknowledge? Maybe the user wants to keep trying to use the wallet-service...